dio/lib/src/form_data.dart

```diff
@@ -30,20 +30,23 @@ class FormData {
   /// Whether [finalize] has been called.
   bool get isFinalized => _isFinalized;
   bool _isFinalized = false;
+  final bool _camelCaseContentDisposition;
 
-  FormData() {
+  FormData({bool camelCaseContentDisposition = false}) {
+    _camelCaseContentDisposition = camelCaseContentDisposition;
```

Comment:
This is invalid since `_camelCaseContentDisposition` is `final`.

Reply:
updated